onyo/lib/inventory.py

```diff
@@ -431,50 +433,68 @@ def get_asset(self, path: Path):
         else:
             raise ValueError(f"{path} is not an asset.")
 
-    def get_assets(self):
-        # plural/query/property?
-        # git ls-files?
-        # Redirect to get_assets_by_query (making paths optional)
-        pass
+    def get_assets(self,
+                   paths: Optional[list[Path]] = None,
+                   depth: int = 0) -> Generator:
+        """Yields all assets up to `depth` directory levels underneath any of `paths`.
```

Comment:
IMO "any of" can be removed. That made me re-read this a few times to parse it.

I propose:

> Yield all assets under `paths` up to `depth` directory levels.